### DISCUSSION TITLE
The new owncast/owncast docker repo

### DISCUSSION THREAD

**Discussion by AMoonRabbit** [owncast/owncast#2691] [Q&A] [answered]
I noted that this new repo currently exists, however lacks ARM support as it's only for amd64? Is this coming soon? I'd like to move over to that image repo but obviously it's incompatible with my HW at the moment.

**Comment by gabek** [comment, marked as answer]
I had pushed the existing tags from the old repo to this new one, but I guess that didn't work. I'll look at it, but it might not be until v0.1.0 is released when I'm back to looking at Docker Hub. Hopefully using the existing repository can continue to work for you in the time being.

**Reply by AMoonRabbit** [reply]
Thanks for the response. Yeah, the current ARM supported repo is working fine.
